commit 881dd5896471df3348f396bdcd3572883d0b8a09
Author: Yves-Alexis Perez <corsac@debian.org>
Date:   Wed Aug 26 20:13:51 2009 +0200

    Revert "Bug #350007 - evolution busy-waits on GPG signing operations"
    
    This reverts commit b2d5dc320e051df875be7847386044e8de33cbbd.

diff --git a/camel/camel-gpg-context.c b/camel/camel-gpg-context.c
index bfd508d..2db58e4 100644
--- a/camel/camel-gpg-context.c
+++ b/camel/camel-gpg-context.c
@@ -498,7 +498,6 @@ gpg_ctx_get_argv (struct _GpgCtx *gpg, gint status_fd, gchar **sfd, gint passwd_
 	g_ptr_array_add (argv, (guint8 *) "--no-secmem-warning");
 	g_ptr_array_add (argv, (guint8 *) "--no-greeting");
 	g_ptr_array_add (argv, (guint8 *) "--no-tty");
-	g_ptr_array_add (argv, (guint8 *) "--no-use-agent");
 
 	if (passwd_fd == -1) {
 		/* only use batch mode if we don't intend on using the
